### PR TITLE
php pi() function entry

### PR DESCRIPTION
**Pull request by arisdelacruz**
### Description

This is my attempt to create a documentation for the pi() function in php. Motivation emanates from the first successful attempt of a creating a documentation for sql.

### Type of Change

- Adding a new entry

### Checklist

- [ /] All writings are my own.
- [ /] My entry follows the Codecademy Docs style guide.
- [ /] My changes generate no new warnings.
- [ /] I have performed a self-review of my own writing and code.
- [ /] I have checked my entry and corrected any misspellings.
- [ /] I have made corresponding changes to the documentation if needed.
- [ /] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ /] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ /] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/math-functions/terms/pi/pi.md
Comment (dakshdeepHERE) [inline content/php/concepts/math-functions/terms/pi/pi.md:38]:
```suggestion
3.1415926535898
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
One change from my side

File: content/php/concepts/math-functions/terms/pi/pi.md
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/pi/pi.md:41]: The codebyte example shouldn't be the same as the prior example. Either eliminate the codebyte or use `pi()` in some sort of calculation.

File: content/php/concepts/math-functions/terms/pi/pi.md
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/pi/pi.md:28]: We need an introductory sentence for the example, even if it's just "The following example prints the value of pi to the console."

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@arisdelacruz I did a first maintainer review and left a few comments.

File: content/php/concepts/math-functions/terms/pi/pi.md
Comment (arisdelacruz) [reply content/php/concepts/math-functions/terms/pi/pi.md:41]: noted on this

will do further edits

File: content/php/concepts/math-functions/terms/pi/pi.md
Comment (arisdelacruz) [reply content/php/concepts/math-functions/terms/pi/pi.md:28]: noted on this

will edit accordingly

**Comment by arisdelacruz** [comment]
@SSwiniarski dunno if i got stuck and i think i need help

were you able to view my edits? 

was instructed by prompt to pull first before pushing again, however, i'm not sure if i need to do another pull request for the said edits

thanks

**Comment by SSwiniarski** [comment]
@arisdelacruz You shouldn't need to do another pull request. You just need to push your commits to the remote branch.

**Comment by dakshdeepHERE** [comment]
> @SSwiniarski dunno if i got stuck and i think i need help
> 
> were you able to view my edits?
> 
> was instructed by prompt to pull first before pushing again, however, i'm not sure if i need to do another pull request for the said edits
> 
> thanks

This happen when you committed changes here online on github website. Which are changes on your repo branch present on website. But did the same changes you did online went back to your pc? No right, then you go back to your pc and make some more changes and try to push them. So now you have 2 changes on same files one changes on web and one on local. That's why git prompt you to pull first. So whenever you commit some changes here make sure you go back on pc and run the command **`git pull branch_name`**.  Then make the changes on pc and push them easily.

**Comment by arisdelacruz** [comment]
Duly noted. Thanks, guys

**Comment by dakshdeepHERE** [comment]
> Duly noted. Thanks, guys

Haha, It's totally okay. I've messed up so many times like this in start. You always learn something new on git when you work more with it. 🙂

**Comment by arisdelacruz** [comment]
@SSwiniarski @dakshdeepHERE 

guys, i just did a git push and i hope that i did the right thing

hehe

thanks

**Review by SSwiniarski — APPROVED** [review]
@arisdelacruz I'm passing this on for a second review.

**Review by caupolicandiaz — APPROVED** [review]
@arisdelacruz @SSwiniarski LGTM

**Comment by arisdelacruz** [comment]
@SSwiniarski @caupolicandiaz thank you, guys

**Comment by arisdelacruz** [comment]
@SSwiniarski BIG THANKS!

**Comment by arisdelacruz** [comment]
@SSwiniarski just wonderin' though

How about some of the other predefined named constants related to pi ?

d'you think we need to include them? I can modify my entry accordingly

thanks

**Comment by arisdelacruz** [comment]
@SSwiniarski the likes of M_1_PI, M_2_SQRTPI, M_PI_4 and many more

**Comment by SSwiniarski** [comment]
@arisdelacruz, I think you could have a section of "Related Constants" with a bulleted list of a few of them.

**Comment by SSwiniarski** [comment]
@arisdelacruz yes, for now.

**Comment by arisdelacruz** [comment]
Ayt, @SSwiniarski 

Thanks

**Pull request merged**
